schema/url/correlationassaymetadata/plugin-correlationassaymetadata-bipartitenetwork.raml

```diff
@@ -6,8 +6,22 @@ types:
   CorrelationAssayMetadataBipartitenetworkPostRequest:
     type: DataPluginRequestBase
     properties:
-      computeConfig: EmptyDataPluginSpec
-      config: EmptyDataPluginSpec
+      computeConfig: CorrelationComputeConfig
+      config: CorrelationAssayMetadataBipartitenetworkSpec
 
   CorrelationAssayMetadataStatsResponse:
-    type: DifferentialAbundanceStatsResponse
+    additionalProperties: false
```

Comment:
is this specific to assay-metadata? or any correlation?

Reply:
also how does this relate to the CorrelationStatsResponse you picked up from compute? what am i missing?

Reply:
updated

Reply:
It's specific to a bipartite network representing correlation data.